app/src/main/java/ch/epfl/sdp/blindly/settings/SettingsShowMe.kt

```diff
@@ -36,8 +39,8 @@ class SettingsShowMe : AppCompatActivity() {
 
         if (currentShowMe != null) {
             displayCheckmark(currentShowMe!!, checks)
-            setOnClickListener(showMeMen, "Men", checks)
-            setOnClickListener(showMeWomen, "Women", checks)
+            setOnClickListener(showMeMen, "Man", checks)

```

Comment:
I don't see why plural form is converted to singular, here we say e.g. "Show me men" in general and not a specific man or a woman so it was more fitting. Same remark for test class as well.

Reply:
I changed it to match the form in ProfileShowMe, but it probably made more sense to do it the other way around (and modify ProfileShowMe...)
We could maybe change all of these in the next refractor (we'll probably need to change the profile setups to fragments)?

Reply:
Yeah grammatically and in terms of what the buttons show ProfileShowMe should have been changed. In the end it's as you wish but this PR changed the other way around and if we leave this to future PR/refactor they can easily become a could-have and we might forgot/miss all the places you have changed.

Reply:
Yep you're right, I think I caught 'em all! ;-)